packages/@aws-cdk/aws-scheduler-targets-alpha/test/lambda-invoke.test.ts

```diff
@@ -85,6 +85,63 @@ describe('schedule target', () => {
     });
   });
 
+  test('creates IAM role and IAM policy for lambda version', () => {
+    const lambdaVersion = new lambda.Version(stack, 'MyLambdaVersion', {
+      lambda: func,
+    });
+    const lambdaTarget = new LambdaInvoke(lambdaVersion, {});
```

Comment:
nothing to do with your PR, but since `LambdaInvoke` can be valid with no properties, we should allow the second field to be optional so we can write `LambdaInvoke(lambdaVersion)`. As simple as:

```ts
  constructor(
    func: lambda.IFunction,
    props: ScheduleTargetBaseProps = {},
  ) {
```